internal/core/metadata/rest_addressable.go

```diff
@@ -35,9 +36,10 @@ func restGetAllAddressables(
 	w http.ResponseWriter,
 	loggingClient logger.LoggingClient,
 	dbClient interfaces.DBClient,
-	errorHandler errorconcept.ErrorHandler) {
+	errorHandler errorconcept.ErrorHandler,
+	config config.ConfigurationStruct) {
```

Comment:
Pass `config.ConfigurationStruct` down the call stack by reference, not by value.

Commented here; occurs elsewhere.

Reply:
- My understanding was that interfaces are passed as references (_by default_). By looking up the online wisdom, this assumption does not seem to be clear cut... Your suggestion, as I reflect on this, is sound. Made it so.

Reply:
`config.ConfigurationStruct` is not an interface.